plugins/@grouparoo/mixpanel/src/lib/test.ts

```diff
@@ -0,0 +1,13 @@
+import { TestPluginMethod } from "@grouparoo/core";
+import { connect } from "./connect";
+
+export const test: TestPluginMethod = async ({ appOptions }) => {
+  const client = await connect(appOptions);
+  const result = await client.query.profile.getSavedCohorts();
```

Comment:
For a test method, will this always work... even if you have no saved cohorts?

Reply:
Yes, it'll return [] (the current case, btw)